src/gravel/api/bootstrap.py

```diff
@@ -89,6 +89,7 @@ async def start_bootstrap(request: Request) -> StartReplyModel:
             message=e.message
         )
     except Exception as e:
+        logger.exception(e)
         logger.error(f"[API] unknown error on bootstrap: {str(e)}")
```

Comment:
(nit) I believe you can just replace the above two lines with:
`logger.exception("[API] unknown error on bootstrap")`
(The exception and original message will also be logged)